website/docs/r/instance_config.html.markdown

```diff
@@ -161,6 +163,22 @@ The following attributes are available on devices:
 
   * `disk_id` - (Optional) The Disk ID to map to this `device` slot
 
+#### device (recommended)
+
+Compared with `devices`, `sda` ... `sdh` is now in the `device_name` attribute in a device block, and the block itself becomes unnamed.
```

Comment:
I think the comparison here could be moved to a deprecation notice under `devices`, and this line could be changed to something along the lines of:

```
An assignment between a disk and a configuration profile device. This block supersedes the `devices` block.
```

It might also be helpful to list the available fields for this block, even though it is a bit redundant.

Reply:
Fix incoming, thank you!